MiniApp/Classes/Storage/MiniAppStatus.swift

```diff
@@ -44,4 +46,26 @@ class MiniAppStatus {
         }
         return nil
     }
+
+    func setCustomPermissions(forMiniApp id: String, permissionList: [MASDKCustomPermissionModel]) {
```

Comment:
I think we shouldn't cache the `description`. This is a dynamic value which could change in the Mini App (i.e. between different Mini App versions, different device language settings, etc.). I think we don't have a requirement for showing this description on the permission revoking screen.

Reply:
Based on our recent discussion, I will be removing the description from the caching.